cf-custom-resources/lib/env-controller.js

```diff
@@ -16,6 +16,8 @@ const updateStackWaiter = {
 
 const AliasParamKey = "Aliases";
 
+let ignoredEnvOutputs = new Set(["EnabledFeatures", "LastForceDeployID"]);
```

Comment:
Can you add a comment w a link to the 4096 bytes limit and why we're ignoring it here? 🥺